core/ir/aarch64/codec.c

```diff
@@ -1323,6 +1323,7 @@ encode_opnd_b_const_sz(uint enc, int opcode, byte *pc, opnd_t opnd, OUT uint *en
     return false;
 }
 
+#if 0  /* Currently unused. */
 /* h_const_sz: Operand size for half (16-bit) vector elements
```

Comment:
If these routines are not exported, isn't it fine to remove them? Or do we think they might be used in the near future?

Reply:
Let's ask the owners: @AssadHashmi the latest clang (14) flags a bunch of codec.c functions as unused -- were these used in the past and are now truly dead, or  are these planned to be used in the future?

Reply:
No reply yet: I guess they can always be restored from version control so I will remove them.

Reply:
Removing them breaks the aarch64_check_codec_order.py script which statically examines this code and compares to opnd_defs.txt.  So we'd have to remove lines from opnd_defs.txt too.  I'm going to put them back and defer to @AssadHashmi on a longer-term fix.